Automattic/simplenote-ios
# Spring Cleaning: New Note Options View

### Fix

This PR adds the new note options button and wires up a basic view with controller and storyboard.

Sub-task of #741.

### Test

1. Select a note from notes list.
2. Confirm the new note options button displays correctly. Currently flagged with a red background.
3. Confirm the view presentation works as expected. Can dismiss with Done button or by swiping down.

**Note:** The various toggles and commands are not wired up yet.

![Note-Options-New](https://user-images.githubusercontent.com/40267301/81569782-4e0b4c80-93d2-11ea-9d3c-6ecde8e76802.png)

PS: @jleandroperez — Nice small changeset for you. You expressed some concern about using storyboards over XIBs so I wanted to make sure this works for you.
